src/Voron/Data/Tables/Table.cs

```diff
@@ -412,6 +415,9 @@ public long Insert(TableValueBuilder builder)
                 page.Flags = PageFlags.Overflow | PageFlags.RawData;
                 page.OverflowSize = size;
 
+                ((RawDataOverflowPageHeader*)page.Pointer)->SectionOwnerHash = Hashing.XXHash64.Calculate(Name.Content.Ptr, (ulong)Name.Content.Length);
```

Comment:
Pretty sure that we can calculate this value one (in fact, it is already there in the `ActiveRawDataSection`, no?